src/errors.ts

```diff
@@ -9,6 +9,13 @@ export interface AzFuncError {
     isAzureFunctionsSystemError: boolean;
 }
 
+export interface TodoError extends Error, Partial<AzFuncError> {
```

Comment:
Please update the name `TodoError` which came from my example haha :)

Reply:
I was a bit unsure about the name, but I guessed you meant setting the error message was something `ToDo` on the object 😅